helsingborg-stad/helsingborg-io-sls-api
# refactor(uploadattachment): store file under unique id only

## Explain the changes you’ve made
Remove the need of filename in body for storing files in S3.

## Explain why these changes are made
The application should only upload files and let the backend store it under its own unique id. The unique upload id is then returned to the client.

## How to test

1. Checkout this branch
2. run `sls deploy` within service folder for `service users-api`
3. Run the endpoint in whatever way you like and check the response
